CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- Change sedge data path from `docker-compose-scripts` to `sedge-data`.e
```

Comment:
This comes under **Changed** not **Added**. Please fix

Reply:
Also please change it to this:
'Rename sedge data directory name to `sedge-data` instead of `docker-compose-scripts`.'

Reply:
Done

Reply:
Here [4fd237d](https://github.com/NethermindEth/sedge/pull/159/commits/4fd237dea72e9057544eb24bfb45a89d900b2ca8)